components/board/jetson/data.go

```diff
@@ -19,306 +19,196 @@ const (
 
 // TODO [RSDK-3596]: fix ngpio numbers in pin definitions for the jetsonTX1, jetsonNano.
 var claraAGXXavierPins = []genericlinux.PinDefinition{
-	{map[int]int{169: 106}, map[int]string{169: "PQ.06"}, "2200000.gpio", 7, 4, "MCLK05", "SOC_GPIO42", "", -1},
-	{map[int]int{169: 112}, map[int]string{169: "PR.04"}, "2200000.gpio", 11, 17, "UART1_RTS", "UART1_RTS", "", -1},
-	{map[int]int{169: 51}, map[int]string{169: "PH.07"}, "2200000.gpio", 12, 18, "I2S2_CLK", "DAP2_SCLK", "", -1},
-	{map[int]int{169: 96}, map[int]string{169: "PP.04"}, "2200000.gpio", 13, 27, "GPIO32", "SOC_GPIO04", "", -1},
+	{Name: "7", Ngpio: 169, LineNumber: 106, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "11", Ngpio: 169, LineNumber: 112, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "12", Ngpio: 169, LineNumber: 51, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "13", Ngpio: 169, LineNumber: 96, PwmChipSysfsDir: "", PwmID: -1},
 	//  Older versions of L4T don"t enable this PWM controller in DT, so this PWM
 	// channel may not be available.
-	{
-		map[int]int{169: 84},
-		map[int]string{169: "PN.01"},
-		"2200000.gpio",
-		15,
-		22,
-		"GPIO27",
-		"SOC_GPIO54",
-		"3280000.pwm",
-		0,
-	},
-	{map[int]int{40: 8, 30: 8}, map[int]string{30: "PBB.00"}, "c2f0000.gpio", 16, 23, "GPIO8", "CAN1_STB", "", -1},
-	{
-		map[int]int{169: 44},
-		map[int]string{169: "PH.00"},
-		"2200000.gpio",
-		18,
-		24,
-		"GPIO35",
-		"SOC_GPIO12",
-		"32c0000.pwm",
-		0,
```

Comment:
:+1: Thanks for cleaning up the formatting on these weird ones, too! Similar code should have similar silhouettes.